isb_lib/models/isb_core_record.py

```diff
@@ -21,21 +21,17 @@ class ISBCoreRecord(SQLModel, table=True):
         description="The time the record was last updated in the iSamples index",
         index=False,
     )
-    source_updated_time: Optional[str] = (
-        Field(
-            default=None,
-            nullable=True,
-            description="The time the record was last updated in the original data, if available",
-            index=False,
-        ),
+    source_updated_time: Optional[str] = Field(
+        default=None,
+        nullable=True,
+        description="The time the record was last updated in the original data, if available",
+        index=False,
```

Comment:
Turns out I had introduced a number of compiler errors when writing this file.  Somehow it still ran and compiled successfully, but the types were all wrong.